crates/board/Cargo.toml

```diff
@@ -21,6 +21,7 @@ aes-gcm = { version = "0.10.3", default-features = false, optional = true }
 bytemuck = { version = "1.16.0", default-features = false, optional = true }
 ccm = { version = "0.5.0", default-features = false, optional = true }
 crypto-common = { version = "0.1.6", default-features = false, optional = true }
+ctap-hid-fido2 = { version = "3.5.1", default-features = false, optional = true }
```

Comment:
This crate does not do what you think it does. It implements the client part of CTAP2, not the authenticator part. I don't think there are any crates. You might have to implement something yourself. You can implement a `UsbClass` directly like https://github.com/google/wasefire/blob/main/crates/protocol-usb/src/device.rs or use https://crates.io/crates/usbd-human-interface-device which might work.

Reply:
I didn't realize that, thank you! I'll re-implement this.

Reply:
No worries, I agree this is quite confusing. Ideally, this should be caught by CI, but due to the conflicts it didn't run. (But maybe because it's WASM, the fact that that crate was not no-std didn't fail.)